plugin/pkg/scheduler/generic_scheduler_test.go

```diff
@@ -397,6 +398,23 @@ func makeNode(node string, milliCPU, memory int64) *v1.Node {
 	}
 }
 
+func TestHumanReadableFitError(t *testing.T) {
+	error := &FitError{
+		Pod: &v1.Pod{ObjectMeta: v1.ObjectMeta{Name: "2"}},
+		FailedPredicates: FailedPredicateMap{
+			"1": []algorithm.PredicateFailureReason{algorithmpredicates.ErrNodeUnderMemoryPressure},
+			"2": []algorithm.PredicateFailureReason{algorithmpredicates.ErrNodeUnderDiskPressure},
+			"3": []algorithm.PredicateFailureReason{algorithmpredicates.ErrNodeUnderDiskPressure},
+		},
+	}
+	if strings.Contains(error.Error(), "No nodes are available that match all of the following predicates") {
```

Comment:
I liked the constant idea, but kept the constant unused in the unit test -- simply because it would make a "self-fulfilling pass" .  The decoupling of unit test from the code is necessary for this feature since otherwise, regression in the readability of the message could go unchecked.